### ISSUE TITLE
INSTALL.md environment incompatibilities

### ISSUE THREAD

**Issue by alexanderbergman7** [facebookresearch/pytorch3d#993]
## 🐛 Bugs / Unexpected behaviors
Following the instructions exactly for installation in INSTALL.md results in conda being unable to solve the environment, and pytorch3d being unable to be installed.

## Instructions To Reproduce the Issue:
In the installation section, I'm able to install the dependencies:
```
conda create -n pytorch3d python=3.9
conda activate pytorch3d
conda install -c pytorch pytorch=1.9.1 torchvision cudatoolkit=10.2
conda install -c fvcore -c iopath -c conda-forge fvcore iopath
conda install -c bottler nvidiacub
```

However, when running the command to install pytorch3d:
```
conda install pytorch3d -c pytorch3d
```

Conda is unable to solve the environment, and a huge amount of conflicts and incompatible packages are found. 

I've found that when downgrading the PyTorch version from 1.9.1 to 1.7.1, then pytorch3d works as expected. But I would like to have a version of PyTorch >1.8.0 in order to use other libraries, and it seems that the current pytorch3d version on Anaconda Cloud does not support this?

**Comment by zhoumeng-hub** [comment]
I also downgrade the PyTorch version from 1.9.1 to 1.7.1 and it works. Thank you so much.

**Comment by bottler** [comment]
There were some builds missing on anaconda for PyTorch3D version 0.6.0 which was the latest when this issue was reported. We have now released 0.6.1 with all its builds and this problem should be fixed.
